forui/lib/src/theme/font/font_data.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:flutter/material.dart';
+
+/// A class that holds the font data.
```

Comment:
```suggestion
/// The default font data that is inherited from a [FTheme] by child Forui widgets.
```

I think we should further explain what the fontSize & other fields are used for and how they are used to scale text in widgets.